Sources/AirwallexRisk/Helpers/Storage.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  Storage.swift
+//  AirwallexRisk
+// 
+//  Created by Richie Shilton on 6/7/2023.
+//  Copyright © 2023 Airwallex. All rights reserved.
+//
+	
+
+import Foundation
+
+@propertyWrapper
+struct Storage<T: Codable> {
+    private let key: String
+    private let defaultValue: T
+    private let defaults: UserDefaults?
+
+    /// Property wrapper to store Codable objects in UserDefaults.
+    /// If no value is set, `defaultValue` will be stored to UserDefaults for subsequent usage.
+    ///
+    init(
+        key: String,
+        defaultValue: T,
+        defaults: UserDefaults? = .sdk
```

Comment:
Would it make more sense to be non-optional?

Reply:
The `.init(suiteName:)` is unfortunately optional